### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/advanced/settings.md`

### PR DESCRIPTION
**Pull request by Jasonw372**
I want to add a translation for the Chinese document

### CODE REVIEW

File: docs/zh/docs/advanced/settings.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+在许多情况下，你的应用程序可能需要一些外部设置或配置，例如密钥、数据库凭据、电子邮件服务的凭据等。
+
+这些设置大多是可变的（可以更改），如数据库的 URLs。而且许多可能是敏感的，如隐私。
+
+由于这个原因，通常是在环境变量中提供它们，再由应用程序读取。
+
+## 环境变量
+
+!!! tip
+如果你已经知道什么是 “环境变量” 以及如何使用它们，请直接跳到下面的章节。
```
Comment (Jedore) [inline docs/zh/docs/advanced/settings.md:12]:
```suggestion
    如果你已经知道什么是 “环境变量” 以及如何使用它们，请直接跳到下面的章节。
```

File: docs/zh/docs/advanced/settings.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+这里，我们在你的 Pydantic `Settings` 类中创建一个 `Config` 类，并将 `env_file` 设置为我们想要使用的 dotenv 文件的文件名。
+
+!!! tip
+使用 `Config` 类仅仅是为了 Pydantic 的设置. 你可以在<a href="https://pydantic-docs.helpmanual.io/usage/model_config/" class="external-link" target="_blank">Pydantic Model Config</a>中了解更多。
+
+### 只用 `lru_cache` 创建 `Settings` 一次
+
+从磁盘上读取文件通常是一个昂贵的（缓慢的）操作，所以你可能只想读取一次，然后重新使用相同的设置对象，而不是为每个请求读取它。
+
+但每次我们都会这样做。
```
Comment (Jedore) [inline docs/zh/docs/advanced/settings.md:288]:
```suggestion
但每次我们都会这样做:
```

**Review by Jedore — CHANGES_REQUESTED** [review]
Good work. Just some little suggestions.

File: docs/zh/docs/advanced/settings.md (hunk 1 above)
Comment (Jedore) [reply docs/zh/docs/advanced/settings.md:12]: This sentence should be contained by `tip`.

**Comment by Jasonw372** [comment]
Thank you @Jedore

**Comment by Jasonw372** [comment]
Please tell me when this translation will be updated to the official website. There are still many documents on the official website that have not been translated into Chinese. Thank you!

**Comment by Jedore** [comment]
> Please tell me when this translation will be updated to the official website. There are still many documents on the official website that have not been translated into Chinese. Thank you!

At least 2 reviewers review your PR.@Jasonw372

**Comment by tiangolo** [comment]
It seems this was translated at some point in another PR. So I'll close this one, but thanks for the effort! ☕

**Pull request closed without merging**